src/test/transaction_tests.cpp

```diff
@@ -98,6 +97,7 @@ BOOST_AUTO_TEST_CASE(tx_valid)
     //
     // verifyFlags is a comma separated list of script verification flags to apply, or "NONE"
     UniValue tests = read_json(std::string(json_tests::tx_valid, json_tests::tx_valid + sizeof(json_tests::tx_valid)));
+    std::string comment("");
```

Comment:
The transaction tests were basically impossible to debug without printing the comments preceding the test.